ssl-impl/src/main/java/io/crate/protocols/ssl/SslConfiguration.java

```diff
@@ -84,7 +85,7 @@ static SslContext buildSslContext(Settings settings) {
                 trustManagers = trustStoreSettings.get().trustManagers;
             }
 
-            SSLContext sslContext = SSLContext.getInstance("TLS");
+            SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
Maybe add a comment why we need that (the browser problem that you mentioned I guess).